app/views/upload.scala.html

```diff
@@ -8,6 +8,19 @@
         @progressIndicator(Messages("upload.progress"))
 
         <h1 class="govuk-heading-xl">@Messages("upload.title")</h1>
+        <div id="uploadError" class="govuk-error-summary hide" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
+        <h2 class="govuk-error-summary__title" id="error-summary-title">
+            @Messages("upload.error.title")
+        </h2>
+        <div class="govuk-error-summary__body">
+            <ul class="govuk-list govuk-error-summary__list">
+                <li>
+                    <p>@Messages("upload.error.failure")</p>
+                    <p class="errorMessage"></p>
```

Comment:
I think by BEM rules, the parent div (the one that currently has an ID of `uploadError` should have a class that's something like `upload-error`, and this element would then be `<p class=upload-error__message>`.

Reply:
Changed